OTAnalytics/plugin_ui/cli.py

```diff
@@ -120,9 +152,24 @@ def parse(self) -> CliArguments:
         """
         args = self._parser.parse_args()
         return CliArguments(
-            args.cli, args.debug, args.ottrks, args.otflow, args.save_name
+            args.cli,
+            args.debug,
+            args.ottrks,
+            args.otflow,
+            args.save_name,
+            self._parse_event_format(args.event_format),
+            args.count_interval,
         )
 
+    def _parse_event_format(self, event_format: str) -> EventFormat:
+        match event_format.lower():
+            case EventFormat.CSV.value:
+                return EventFormat.CSV
+            case EventFormat.EXCEL.value:
+                return EventFormat.EXCEL
+            case _:
+                return EventFormat.OTEVENTS
+
 
```

Comment:
If we parse the input and directly return the required `EventlistExporter` we do not need the `match ... case ...` statement when exporting.